Update the evaluation license troubleshooting section (#486)
MicroEJ/docs: SDKUserGuide/licenses.rst

   Invalid License Key Error Message

Machine UID has changed
"""""""""""""""""""""""

> This can occur when the hardware configuration of the machine is changed (especially when the network interfaces have changed).

Reviewer: ```suggestion
This can occur when the hardware configuration of the machine has changed such as network interfaces.
```
